workers/data_refinery_workers/processors/array_express.py

```diff
@@ -23,16 +23,23 @@ def cel_to_pcl(kwargs: Dict):
     # Array Express processor jobs have one batch per job.
     batch = kwargs["batches"][0]
 
-    from_directory = utils.ROOT_URI + "raw/" + batch.internal_location
-    target_directory = utils.ROOT_URI + "processed/" + batch.internal_location
+    raw_file = os.path.join(utils.ROOT_URI, "raw", batch.internal_location, batch.name)
+
+    target_directory = os.path.join(utils.ROOT_URI, "processed", batch.internal_location)
     os.makedirs(target_directory, exist_ok=True)
-    new_name = batch.name + "." + batch.processed_format
 
-    ro.r('source("/home/user/r_processors/process_cel_to_pcl.R")')
-    ro.r['ProcessCelFiles'](
-        from_directory,
-        "Hs",  # temporary until organism handling is more defined
-        target_directory + new_name)
+    processed_file = os.path.join(target_directory, batch.name.split(".")[0])
```

Comment:
If there is meaningful structure in the `batch.name` property, do we want to rely on the code here in the `array_express` processor to understand that structure (and stay in sync with it as modified), or would it be better to define a method on `batch` for this such as `batch.get_filename()`?